roles/ceph-osd/defaults/main.yml

```diff
@@ -255,6 +255,14 @@ ceph_config_keys: [] # DON'T TOUCH ME
 ceph_osd_docker_memory_limit: 1g
 ceph_osd_docker_cpu_limit: 1
 
+# The next two variables are undefined, and thus, unused by default.
```

Comment:
please run  `generate_group_vars_sample.sh`

Reply:
OK, I did that and included the generated group_vars/osds.yml.sample in the updated PR